docs/changelog.rst

```diff
@@ -425,6 +425,9 @@ Bug fixes:
   `requests` timeout.
 * Fix cover art resizing logic to support multiple steps of resizing
   :bug:`5151`
+* Fix bug where matcher doesn't consider medium number when importing making
```

Comment:
One last thing - you probably want to rebase off the most recent `master` version and move your changelog note to the top, under the **Unreleased** section!

Reply:
Ok, rebased and squashed. Changelog note moved to the top unreleased section.